spec/models/service_template_ansible_tower_spec.rb

```diff
@@ -87,6 +87,27 @@
       expect(updated.resource_actions.pluck(:action)).to match_array(%w(Provision Reconfigure))
     end
 
+    it 'does not allow service_type to be changed' do
```

Comment:
@bzwei ok - just thought it would be a good idea to ensure that it includes the super class method call